index.bs

```diff
@@ -514,4 +514,47 @@ See [[FS#sandboxed-filesystem]].
 
 </aside>
 
+<h3 id="storage-bucket-clear-site-data">Clear Site Data integration</h3>
+
+Issue: Update [[clear-site-data#header]].
+
+: "<dfn grammar>`storage:bucket-name`</dfn>"
+
+  ::  If the type string starts with "`storage:`" then the remaining characters after the
+      `:` will be taken to refer to a specific [=storage bucket=] in |origin|.
+
+Issue: Update [[clear-site-data#parsing]] with these partial algorithms.
+
+<div algorithm>
+
+To <dfn>parse a Clear-Site-Data header with buckets</dfn>, execute the following steps:
+
+1. For each |type| in |header|, execute the following steps:
```

Comment:
Should `|header|` be defined before this, maybe like CSD?
`Let |header| be the result of extract header list values...` [link](https://w3c.github.io/webappsec-clear-site-data/#parsing)